sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_resource_configuration.py

```diff
@@ -124,7 +124,7 @@ def __init__(
         locations: Optional[List[str]] = None,
         instance_count: Optional[int] = None,
         instance_type: Optional[Union[str, List]] = None,
-        properties: Optional[Properties] = None,
+        properties: Optional[Union[Properties, Dict]] = None,
```

Comment:
The original was `Dict[str, Any]`, so can we expand this to `Optional[Union[Properties, Dict[str, Any]]]`?

cc: @annatisch